Cakes-Co/index.html

```diff
@@ -1,16 +1,129 @@
+      <img
+        src="/Cakes-Co/img/cake logo.png"
+        class="cupcake-logo"
+        alt="cake-picture"
+      />
+
+      <p>Get the finest cakes delivered to your door</p>
+    </header>
+
+    <!-- NAVIGATION SECTION -->
+    <label for="burger" class="burger-icon">&#9776;</label>
+    <nav>
+      <input type="checkbox" class="checkbox-burger" id="burger" />
+      <div class="flex-nav">
+        <a href="#" class="home">HOME</a>
+        <a href="#" class="cakes">CAKES</a>
+        <a href="#" class="order-in">ORDER IN</a>
+        <a href="#" class="lessons">LESSONS</a>
+        <a href="#" class="about">ABOUT</a>
+      </div>
+    </nav>
+
+    <!-- MAIN SECTION -->
+    <main>
+      <img
+        src="/Cakes-Co/img/Maincake.png"
+        class="main-image"
+        alt="cake-picture-1"
+      />
+      <section>
+        <h1>Welcome</h1>
+        <p class="welcome-text">
+          Welcome to our delightful world of confectionery! At Cakes-Co, we are
+          passionate about turning ordinary moments into extraordinary memories
+          with our scrumptious cakes. Whether you're celebrating a special
+          occasion or simply satisfying your sweet cravings, our delectable
+          creations are here to make every bite a moment to savor. Indulge in a
+          symphony of flavors, from classic chocolate and vanilla to exotic
+          fruit-infused delights. Our expert bakers craft each cake with love
+          and precision, ensuring every slice is a mouthwatering masterpiece.
+          Explore our diverse range of designs, sizes, and customization options
+          to suit your unique preferences. Join us in the sweet journey of
+          celebrations and pure indulgence. Experience the magic of Cakes-Co
+          cakes today!
+        </p>
+      </section>
+    </main>
 
-</html>
\ No newline at end of file
+    <!-- CONTENT SECtION -->
+    <content>
+      <figure>
+        <img
+          src="/Cakes-Co/img/cake1.png"
+          width="100%"
+          height="100%"
+          class="cake-img-1"
+          alt="cake-picture-1"
+        />
+      </figure>
+      <figure>
+        <img
+          src="/Cakes-Co/img/cake2.png"
+          width="100%"
+          height="100%"
+          class="cake-img-2"
+          alt="cake-picture-2"
+        />
+      </figure>
+      <figure>
+        <img
+          src="/Cakes-Co/img/cake3.png"
+          width="100%"
+          height="100%"
+          class="cake-img-3"
+          alt="cake-picture-3"
+        />
+      </figure>
+      <figure>
+        <img
+          src="/Cakes-Co/img/cake14.png"
+          width="100%"
+          height="100%"
+          class="cake-img-4"
+          alt="cake-picture-4"
+        />
+      </figure>
+    </content>
+
+    <!-- FOOTER SECTION -->
+
+    <footer>
+      <div class="contact-info">
+        <div class="Telephone">TELEPHONE</div>
+        <div class="E-mail">E-MAIL</div>
+        <div class="address">ADDRESS</div>
```

Comment:
Consider using the `<a> `tag for your links instead of `<div>`. It provides better usability and accessibility. Example:
```
<a href="tel:+1234567890">TELEPHONE</a>
<a href="mailto:example@example.com">E-MAIL</a>
<a href="https://example.com/address">ADDRESS</a>